app/presenters/notes_presenter.rb

```diff
@@ -20,6 +21,25 @@ def self.synthetic_notes_from_documents(documents)
     end
   end
 
+  def self.synthetic_notes_from_outbound_calls(outbound_calls)
+    outbound_calls.map do |outbound_call|
+      next if outbound_call.twilio_status == "queued"
+      duration = ""
+      if outbound_call.call_duration
+        duration_minutes = outbound_call.call_duration&.to_i / 60
+        duration_seconds = outbound_call.call_duration&.to_i % 60
+        duration += "#{duration_minutes}m" if duration_minutes > 0
+        duration += "#{duration_seconds}s"
```

Comment:
i'm curious whether there's some ruby function that would allow you to just get the formatted string from the number...